platform/quality-assurance/auth-session-loadtest.md

```diff
@@ -0,0 +1,36 @@
+# Using authenticated sessions for load testing
+
+## Things you need
+1. [SOCKS proxy access](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/internal-tools/)
+2. [Access to internal tools](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/internal-tools/) (specifically Jenkins, but if you have access to one then you tend to have access to all)
```

Comment:
fyi: in Markdown, you can use "1." for all your numbers and it'll auto-number them correctly (easier for editing in the future, don't have to change all the numbers if you insert/delete a step in the middle)

Reply:
Excellent -- I was just wondering if that was best practice for MD yesterday.